taskcluster/test-training_upstream-linux-amd64-py27mu-opt.yml

```diff
@@ -1,7 +1,7 @@
 build:
   template_file: test-linux-opt-base.tyml
   dependencies:
-    - "linux-amd64-ctc-opt"
+    - "linux-amd64-cpu-opt"
```

Comment:
I don't think it's a good idea, completion time of `linux-amd64-cpu-opt` is much higher, it's going to delay everything behind, we should re-use `linux-amd64-ctc-opt` to build the new python wheel instead

Reply:
The problem is that it's not a new Python wheel, it's several Python wheels, one for each architecture/Python version, so I tried to leverage the existing infrastructure for CPU builds.

Reply:
Sure, but it's still better IMHO to do that outside, we can easily hack around `do_deepspeech_python_build`